postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -269,13 +269,17 @@ def _get_new_pg_stat_activity(self, available_activity_columns):
         report_activity = self._report_activity_event()
         cur_time_func = ""
         blocking_func = ""
+        backend_type_predicate = "false"
+        if self._check.version.compare(V10) >= 0:
+            backend_type_predicate = "backend_type != 'client backend'"
```

Comment:
what types of things fall under the "client backend" type?

Reply:
> Type of current backend. Possible types are autovacuum launcher, autovacuum worker, logical replication launcher, logical replication worker, parallel worker, background writer, client backend, checkpointer, archiver, startup, walreceiver, walsender and walwriter. In addition, background workers registered by extensions may have additional types.

Reply:
https://www.postgresql.org/docs/15/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW